samples/openapi3/server/petstore/spring-boot-oneof/src/main/java/org/openapitools/model/CreateParkAnimalCarerPersonResponsibleForOneOf.java

```diff
@@ -0,0 +1,34 @@
+package org.openapitools.model;
+
+import java.net.URI;
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
This is the annotation which is missing without my fix, and which is the cause of the bug. This sample proves the code is generated correctly.